ci/dictionary.txt

```diff
@@ -103,3 +103,4 @@ wakeups
 webpages
 webserver
 Woot
+JoinHandle
```

Comment:
It looks like this file is in alphabetical order. Would you mind moving JoinHandle up to about line 40?